content/docs/04.5-devx/05-app-profile.md

```diff
@@ -14,13 +14,13 @@ import Tooltip from "shared/components/ui/Tooltip";
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
```

Comment:
Mention that in the code editor type {{.spectro.}} to see the list available variables in that particular tier.

Reply:
![image](https://user-images.githubusercontent.com/49595451/199450815-8f03f47a-66b7-4dc1-bc3d-6b2449022d9a.png)